packages/jest-cli/package.json

```diff
@@ -39,6 +40,9 @@
     "which": "^1.2.12",
     "yargs": "^11.0.0"
   },
+  "optionalDependencies": {
+    "why-is-node-running": "^2.0.2"
```

Comment:
this needs Node 8 (uses async hooks), hoping sticking it in `optionalDependencies` is enough

Reply:
if not, we could do the same thing `detectLeaks` does - require the user to install it themselves

Reply:
What was the reason we didn't put `weak` in optional dependencies? Package managers tried to install it anyway?

Reply:
The output was horrendous when it failed, and windows installs still crashed. See #5169